scripts/dev_setup.sh

```diff
@@ -480,9 +480,9 @@ function install_z3 {
      return
   fi
   if [[ "$(uname)" == "Linux" ]]; then
-    Z3_PKG="z3-$Z3_VERSION-x64-ubuntu-16.04"
+    Z3_PKG="z3-$Z3_VERSION-x64-glibc-2.31"
```

Comment:
Does the ubuntu version not longer exist? glib is perhaps too generic?

Reply:
An Ubuntu binary exists for the version 4.8.10 (i.e., "z3-4.8.10-x64-ubuntu-18.04.zip"). However, for the versions 4.8.11 and 4.8.12, no Ubuntu-specific binary exists. It seems that "z3-4.8.*-x64-glibc-2.31.zip" replaces it. See [the release page](https://github.com/Z3Prover/z3/releases).

I can't find the name change from "ubuntu" to "glibc" , but [here](https://github.com/Z3Prover/z3/commit/39af2a188da198b87037fe4fad2bd5da67386c86) the name "centos" was changed to "glibc" as a linux binary.

Reply:
Seems that all linux-based releases for z3 are using the `glibc` annotation now.